fizzbuzz.js

```diff
@@ -1,11 +1,99 @@
 // This is our main function
 function fizzbuzz() {
-    console.log("Hello, World!");
 
-    // Put your code here...
+    for (let i = 1; i <= 100; i++) {
+        output = "";
+
+        if (i % 3 === 0) {
+            output = "Fizz";
+        }
+
+        if (i % 5 === 0) {
+            output += "Buzz";
+        }
+
+        console.log(output !== "" ? output : i)
+    }
+
+}
+
+function fizzbuzz2(num) {
+
+    for (let i = 1; i <= num; i++) {
+
+        console.log(getOutput(i))
+    }
+
+}
+
+
+function getOutput(i) {
```

Comment:
Can you think about how you could name these functions to be more descriptive of what they're doing? It's not clear to me why `fizzbuzz1` would loop over `num` and call `console.log(getOutput(i))` for each